src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/NoticeOfChangeController.java

```diff
@@ -74,6 +75,25 @@ public ResponseEntity<CCDCallbackResponse> updateNocRespondents(@RequestBody Cal
             return ResponseEntity.status(FORBIDDEN.value()).build();
         }
 
+        try {
+            nocNotificationService.sendNotificationOfChangeEmails(callbackRequest,
+                    callbackRequest.getCaseDetails().getCaseData());
+        } catch (Exception exception) {
+            log.error(exception.getMessage(), exception);
+        }
+
+        ChangeOrganisationRequest changeOrganisationRequestField =
+            callbackRequest.getCaseDetails().getCaseData().getChangeOrganisationRequestField();
+
+        if (changeOrganisationRequestField != null) {
+            try {
+                nocRespondentRepresentativeService.removeOrganisationRepresentativeAccess(
+                    callbackRequest.getCaseDetails().getCaseId(), changeOrganisationRequestField);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
Wanted to flag this but I believe you're creating a bug ticket to track this separately?

Reply:
https://tools.hmcts.net/jira/browse/RET-3203